tests/roxctl/bats-tests/cluster/centraldb-generate.bats

```diff
@@ -11,6 +11,7 @@ setup_file() {
   command -v grep || skip "Command 'grep' required."
   [[ -n "${API_ENDPOINT}" ]] || fail "Environment variable 'API_ENDPOINT' required"
   [[ -n "${ROX_PASSWORD}" ]] || fail "Environment variable 'ROX_PASSWORD' required"
+  bats_require_minimum_version 1.5.0 # FIXME: Remove before merging
```

Comment:
Sure about the FIXME? It would be cool to keep it (I hope our CI has new bats version that supports this).

Reply:
Looks like our CI is [running bats 1.10](https://github.com/stackrox/rox-ci-image/blob/4093e6917ce8628daf8400f0077a04a28c58d3da/images/stackrox-test.Dockerfile#L65), so looks like this is a keeper 🎉